/*
 * This file was automatically generated by EvoSuite
 * Mon Jul 18 15:44:04 GMT 2016
 */

package org.objectweb.cjdbc.common.exceptions;

import org.junit.Test;
import static xcorpus.AssertionAdapter.*;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;
import org.objectweb.cjdbc.common.exceptions.VirtualDatabaseException;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true) 
public class VirtualDatabaseException_ESTest extends VirtualDatabaseException_ESTest_scaffolding {

  //Test case number: 0
  /*
   * 2 covered goals:
   * Goal 1. org.objectweb.cjdbc.common.exceptions.VirtualDatabaseException.<init>(Ljava/lang/String;)V: root-Branch
   * Goal 2. org.objectweb.cjdbc.common.exceptions.VirtualDatabaseException.<init>(Ljava/lang/String;Ljava/lang/Throwable;)V: root-Branch
   */

	@Test(timeout=300000)
  public void test0()  throws Throwable  {
      VirtualDatabaseException virtualDatabaseException0 = new VirtualDatabaseException("B)`pV");
      VirtualDatabaseException virtualDatabaseException1 = new VirtualDatabaseException("B)`pV", (Throwable) virtualDatabaseException0);
      assertFalse(virtualDatabaseException1.equals((Object)virtualDatabaseException0));
  }

  //Test case number: 1
  /*
   * 2 covered goals:
   * Goal 1. org.objectweb.cjdbc.common.exceptions.VirtualDatabaseException.<init>(Ljava/lang/String;)V: root-Branch
   * Goal 2. org.objectweb.cjdbc.common.exceptions.VirtualDatabaseException.<init>(Ljava/lang/Throwable;)V: root-Branch
   */

	@Test(timeout=300000)
  public void test1()  throws Throwable  {
      VirtualDatabaseException virtualDatabaseException0 = new VirtualDatabaseException("B)`pV");
      VirtualDatabaseException virtualDatabaseException1 = new VirtualDatabaseException((Throwable) virtualDatabaseException0);
      assertFalse(virtualDatabaseException1.equals((Object)virtualDatabaseException0));
  }

  //Test case number: 2
  /*
   * 1 covered goal:
   * Goal 1. org.objectweb.cjdbc.common.exceptions.VirtualDatabaseException.<init>()V: root-Branch
   */

	@Test(timeout=300000)
  public void test2()  throws Throwable  {
      VirtualDatabaseException virtualDatabaseException0 = new VirtualDatabaseException();
  }
}
